docs/pages/extensibility/providers.md

```diff
@@ -4,21 +4,22 @@ title: "Extensibility: Providers"
 permalink: extensibility/providers
 type: Extensibility
 #redirect_from: architecture/adapters
-abstract: "Meshery uses adapters to enrich the level of depth by which it manages cloud native infrastructure."
+abstract: "Meshery uses providers to enrich the level of depth by which it manages cloud native infrastructure."
 language: en
 list: include
 ---
 
-Meshery offers Providers as a point of extensibility. With a built-in Local Provider (named "None"), Meshery Remote Providers are designed to be pluggable. Remote Providers offer points of extension to users / integrators to deliver enhanced functionality, using Meshery as a platform. A specific provider can be enforced in a Meshery instance by passing the name of the provider with the env variable PROVIDER.
+Meshery offers _Providers_ as a point of extensibility. It has a built-in Local Provider, named **"None"** and supports _Remote Providers_ that are designed to be pluggable. _Remote Providers_ offer a point of extension to users / integrators to deliver enhanced functionality, using Meshery as a platform. A specific provider can be enforced in a Meshery instance by passing the name of the provider with the env variable `PROVIDER`.
```

Comment:
The last sentence in this paragraph is worth pulling out and putting into its own section with a little more explanation and an example. We take advantage of this setting in the Playground deployments. Why? So that we secure the public-facing Meshery UI behind a Remote Provider that enforces identity with authN and authZ.

Reply:
Added these changes to the "remote provider" section.